decentraland/redux/reducers.js

```diff
@@ -21,90 +21,105 @@ import { removeFromList, updateListItem, toListIfNot } from "@helpers";
+    }
+  };
+}
+
+//
+// accountInfo reducer
+//
+const setAccount = (state, action) => {
+  const { account } = action;
+  return { ...state, account };
+};
+const setAccountCreationStatus = (state, action) => {
+  const { creationStatus } = action;
+  return { ...state, creationStatus };
+};
+const updateActionIdForAccountCreationStatus = (state, action) => {
+  const { creationStatusAction } = action;
+  const { status, actionId } = creationStatusAction;
+  let { creationActions } = state;
+  creationActions = { ...creationActions, [status]: actionId };
+  return { ...state, creationActions };
+};
+const setAccountCreationActions = (state, action) => {
+  const { creationActions } = action;
+  return { ...state, creationActions };
+};
+
+const accountInfo = createReducer(
+  {
     account: null,
     creationStatus: NOT_STARTED,
     creationActions: {},
   },
-  action
-) {
-  const {
-    type,
-    account,
-    creationStatus,
-    creationStatusAction,
-    creationActions,
-  } = action;
-  switch (type) {
-    case SET_ACCOUNT:
-      return { ...state, account };
-    case SET_ACCOUNT_CREATION_STATUS:
-      return { ...state, creationStatus };
-    case UPDATE_ACTION_ID_FOR_ACCOUNT_CREATION_STATUS: {
-      const { status, actionId } = creationStatusAction;
-      let { creationActions } = state;
-      creationActions = { ...creationActions, [status]: actionId };
-      return { ...state, creationActions };
-    }
-    case SET_ACCOUNT_CREATION_ACTIONS: {
-      return { ...state, creationActions };
-    }
-    default:
-      return state;
+  {
+    [SET_ACCOUNT]: setAccount,
+    [SET_ACCOUNT_CREATION_STATUS]: setAccountCreationStatus,
+    [UPDATE_ACTION_ID_FOR_ACCOUNT_CREATION_STATUS]: updateActionIdForAccountCreationStatus,
+    [SET_ACCOUNT_CREATION_ACTIONS]: setAccountCreationActions,
   }
-}
+);
 
-function selectedLandToBuy(state = null, action) {
-  const { type, landForSale } = action;
-  switch (type) {
-    case SELECT_LAND_TO_BUY:
-      return landForSale;
-    default:
-      return state;
-  }
-}
+//
+// selectedLandToBuy reducer
+//
+const selectLandToBuy = (state, action) => {
+  const { landForSale } = action;
+  return landForSale;
+};
 
-function assetsForSale(state = { list: [], loadingInProgress: true }, action) {
-  const { type, landForSale, loadingInProgress } = action;
-  switch (type) {
-    case PREPEND_LAND_FOR_SALE_TO_LIST:
-      return { ...state, list: [landForSale, ...state.list] };
-    case APPEND_LAND_FOR_SALE_TO_LIST:
-      return { ...state, list: [...state.list, landForSale] };
-    case REMOVE_LAND_FOR_SALE: {
-      let { list: assetsForSale } = state;
-      const list = removeFromList(assetsForSale, landForSale);
-      return { ...state, list };
-    }
-    case SET_LOADING_ASSETS_FOR_SALE_IN_PROGRESS:
-      return { ...state, loadingInProgress };
-    default:
-      return state;
-  }
-}
+const selectedLandToBuy = createReducer(null, {
+  [SELECT_LAND_TO_BUY]: selectLandToBuy,
```

Comment:
The past tense for one name (selected) - the wrapped reducer - and present tense internal function (select) is a little confusing. Maybe let's pick one or the other?

Reply:
`selectedLandToBuy` is the reducer/state that stores the selected land from user
`selectLandToBuy` is the action to select
Maybe change `selectedLandToBuy` to `landToBuy`?

Reply:
Yeah, I like that suggestion. Thanks!

Reply:
Done.

Reply:
Thanks. Resolving this.